api/api/spec/spec.yaml

```diff
@@ -8028,35 +8438,33 @@ paths:
                   totalItems: 18
                   items:
                   - local:
-                      ip: "::"
-                      port: 22
+                      ip: 172.17.0.100
+                      port: 1516
                     remote:
-                      ip: "::"
-                      port: 0
+                      ip: 172.17.0.101
+                      port: 54550
                     scan:
-                      id: 2112182802
-                      time: "2019/03/08 08:30:59"
-                    state: "listening"
-                    tx_queue: 0
+                      id: 1672337392
+                      time: "2019/03/14 10:38:21"
```

Comment:
Example date-times must fit the ISO8601.

Reply:
Done in 6696debbd6e2e0e604ff9663e86f6b69f91b7160